src/sys/mman.rs

```diff
@@ -213,6 +223,14 @@ pub unsafe fn munlock(addr: *const c_void, length: size_t) -> Result<()> {
     Errno::result(libc::munlock(addr, length)).map(drop)
 }
 
+pub unsafe fn mlockall(flags: MlockAllFlags) -> Result<()> {
```

Comment:
Why are these functions `unsafe`?

Reply:
You're right, I suppose there's nothing actually unsafe here. Done.